src/style.css

```diff
@@ -0,0 +1,336 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.flex {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+body {
+  font-family: 'Courgette', cursive, monospace, sans-serif, serif;
+  overflow-x: hidden;
+  margin: 0;
+  min-height: 100vh;
+}
+
+.max-w {
+  max-width: 1200px;
+  margin: auto;
+}
+
+header {
+  padding: 0 10px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100%;
+}
+
+.logo {
+  width: 100px;
+  height: auto;
+}
+
+.nav-items {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  gap: 50px;
+}
+
+.nav-link {
+  list-style-type: none;
+}
+
+.kb-hero-b {
+  width: 100vw;
+  height: 35vh;
+  background-image: url("../assets/kb-hero.png");
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.container {
+  flex-wrap: wrap;
+  align-content: center;
+  gap: 20px;
+  padding: 20px 0;
+}
+
+.card {
+  width: 350px;
+  height: 460px;
+  flex-direction: column;
+  border: 1px solid gainsboro;
+  border-radius: 15px;
+  gap: 0;
+  margin: 0 5px;
+  background-color: gainsboro;
+  transition: all 0.2s ease-in-out;
+}
+
+.card:hover {
+  border: 1px solid yellow;
+  transform: scale(1.03);
+}
+
+.card h1 {
+  text-align: center;
+  font-size: 20px;
+  height: 56px;
+}
+
+.card h6 {
+  color: rgb(189, 184, 184);
+}
+
+.card img {
+  width: 200px;
+  border-radius: 15px;
+  margin: 5px;
+}
+
+.likes {
+  width: 60%;
+  padding: 10px 0;
+  justify-content: space-around;
+}
+
+.card .likes img {
+  width: 30px;
+  transition: all 0.2s ease-in-out;
+}
+
+.card .likes img :hover {
+  transform: scale(1.05);
+}
+
+.crd-btns {
+  gap: 17px;
+}
+
+.card button {
+  border: 1px solid #ffeb3b;
+  border-radius: 5px;
+  height: 40px;
+  width: 90px;
+  background: #ff9800;
+  color: rgb(3, 3, 3);
+  margin: 5px 0;
+}
+
+.card button:hover {
+  border: 1px solid #020202;
+  border-radius: 5px;
+  height: 40px;
+  width: 90px;
+  background: #ffeb3bb9;
+  color: #000;
+  margin: 5px 0;
+}
+
+footer {
+  height: 50px;
+}
+
+footer h3 {
+  text-align: center;
+}
+
+/* Reservation */
+
+.Reservation-modal {
+  width: 80vw;
+  background: rgba(0, 0, 0, 0.85);
+  z-index: 99;
+  position: fixed;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%) scale(0.4);
+  text-align: center;
+  padding: 30px;
+  display: block;
+
+  /* visibility: hidden; */
+  animation-name: modalOpen;
+  animation-duration: 2s;
+  transition: transform 0.4s, top 0.4s;
+  color: #fff;
+  font-size: 24px;
+  border-radius: 3%;
+}
+
+@keyframes modalOpen {
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+}
+
+#closeBtn {
+  width: 84px;
+  height: 84px;
+  position: absolute;
+  z-index: 99;
+  right: 1.3rem;
+  margin-top: 1rem;
+  margin-left: 1.3rem;
+
+  /* background-color: #fff; */
```

Comment:
- [ ] Please do not leave commented code that is not descriptive in your codebase, this would give a cleaner and more professional code.

Reply:
Sure, will be removing that. Thank you